source/uwp/Renderer/lib/AdaptiveColumn.cpp

```diff
@@ -41,7 +41,14 @@ namespace AdaptiveNamespace
 
     HRESULT AdaptiveColumn::get_Width(_Outptr_ HSTRING* width) { return m_width.CopyTo(width); }
 
-    HRESULT AdaptiveColumn::put_Width(_In_ HSTRING width) { return m_width.Set(width); }
+    HRESULT AdaptiveColumn::put_Width(_In_ HSTRING width)
+    {
+        RETURN_IF_FAILED(m_width.Set(width));
+
+        std::vector<std::shared_ptr<AdaptiveCardParseWarning>> warningvector;
```

Comment:
seems this is not going to be used at all, will it be possible not to allocate it at all?

Reply:
I can change ParseSizeForPixelSize to take a pointer instead of a reference and pass null.

---
In reply to: [252409077](https://github.com/Microsoft/AdaptiveCards/pull/2346#discussion_r252409077) [](ancestors = 252409077)

Reply:
Actually, how strongly do we feel about these warnings? It occurs to me i should make this change in the shared model as well, but if i move the logic there from Deserialize() to SetWidth() we won't have the warnings there either. Certainly i could code around that, but i want to make sure that we actually need those warnings before i make the code more complicated than i need to. We're not super consistent with warnings already, as it looks like we don't return a warning if they set the width to, for example, "50" (no "px").

---
In reply to: [252427457](https://github.com/Microsoft/AdaptiveCards/pull/2346#discussion_r252427457) [](ancestors = 252427457,252409077)

Reply:
Alright, reorganized it leaving the warnings in place using a pointer rather than a reference.

---
In reply to: [252430493](https://github.com/Microsoft/AdaptiveCards/pull/2346#discussion_r252430493) [](ancestors = 252430493,252427457,252409077)

Reply:
Indeed